### PR TITLE
Make console logging be enabled/disabled with log levels and scoped loggers

### PR DESCRIPTION
**Pull request by jwforres**

### CODE REVIEW

**Comment by jwforres** [comment]
@liggitt review please

File: assets/app/index.html [hunk 1]
```diff
@@ -111,6 +111,7 @@
         <!-- build:js({.tmp,app}) scripts/scripts.js -->
         <script src="scripts/app.js"></script>
         <script src="scripts/misc/labelSelector.js"></script>
+        <script src="scripts/services/logger.js"></script>
```
Comment (liggitt) [inline assets/app/index.html]: missed checking in the new file?

**Comment by liggitt** [comment]
LGTM other than the new file being missing

File: assets/app/scripts/services/logger.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+angular.module('openshiftConsole')
+.provider('Logger', function() {
+  this.$get = function() {
+    var OSLogger = Logger.get("OpenShift");
```
Comment (liggitt) [inline assets/app/scripts/services/logger.js]: comment that this is using the global `Logger` object... it's a little confusing... I thought it was using the object defined by this service

**Comment by liggitt** [comment]
suggested comment, LGTM otherwise

**Comment by jwforres** [comment]
[merge]

**Pull request merged**